js/tests/unit/tooltip.js

```diff
@@ -680,6 +680,42 @@ $(function () {
     $styles.remove()
   })
 
+  test('should not be adjust position when scrolling', function () {
```

Comment:
@cvrebert this specific test is more about "scrolling should not affect tooltip positioning".  I probably had "readjust" in mind when I typed "be adjust"